### DISCUSSION TITLE
Backing up my db to another server

### DISCUSSION THREAD

**Discussion by ocean-haiyang** [usememos/memos#3920] [Q&A] [answered]
But it only shows the memo entries from a day ago. I see there are two other .db-something files. Does the SQLite db somehow delay writes?

Just trying to back it up frequently

**Comment by RoccoSmit** [comment, marked as answer]
You will need to back up all 3 files. See documentation of what files are here https://www.sqlite.org/walformat.html

Snippet from the above link that could explain your issue:

> The write-ahead log or "wal" file is a roll-forward journal that records transactions that have been committed but not yet applied to the main database

**Reply by ocean-haiyang** [reply]
> You will need to back up all 3 files. See documentation of what files are here https://www.sqlite.org/walformat.html
> 
> Snippet from the above link that could explain your issue:
> 
> > The write-ahead log or "wal" file is a roll-forward journal that records transactions that have been committed but not yet applied to the main database

Thank you! I was confused because the new memos instance only had the .db file. You are correct.
